contracts/crosschain/README.adoc

```diff
@@ -0,0 +1,38 @@
+= Crosschain
+
+[.readme-notice]
+NOTE: This document is better viewed at https://docs.openzeppelin.com/community-contracts/proxy
+
+Gateways are contracts that enable cross-chain communication. These can either be a message source or a destination according to ERC-7786, which defines the following interfaces:
+
+ * {IERC7786GatewaySource}: A contract interface to send a message to another contract on a different chain.
+ * {IERC7786Receiver}: An interface that allows an smart contract to receive a crosschain message provided by a trusted destination gateway.
+
+The library provides an implementation of an ERC-7786 receiver contract as a building block:
+
+ * {ERC7786Receiver}: ERC-7786 cross-chain message receiver.
+
+Given ERC-7786 could be enabled natively or via adapters. Developers can access interoperability protocols through gateway adapters. The library includes the following gateway adapters:
+
+ * {AxelarGatewayBase}: Gateway adapter implementation for the https://www.axelar.network/[Axelar Network].
+ * {AxelarGatewaySource}: ERC-7786 sender gateway. It sends a cross-chain message through Axelar.
+ * {AxelarGatewayDestination}: ERC-7786 destination gateway. It executes a cross-chain message from Axelar.
+ * {AxelarGatewayDuplex}: ERC-7786 gateway that operates in both directions (i.e. send and receive messages) using the Axelar network.
+
+== Gateways
+
+{{IERC7786GatewaySource}}
+
+{{IERC7786Receiver}}
+
+{{ERC7786Receiver}}
```

Comment:
`IERC7786Receiver` and `ERC7786Receiver` are not Gateways, there are clients of gateways 
```suggestion
== Clients

{{IERC7786Receiver}}

{{ERC7786Receiver}}
```